src/pages/tables/admins-table.html

```diff
@@ -0,0 +1,778 @@
+              <i class="menu-icon mdi mdi-file-document-multiple-outline"></i>
+              <span class="menu-title">Articles</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <!------------------------------------------------Event Category--------------------------------------------------->
+          <li class="nav-item nav-category">Event</li>
+          <li class="nav-item">
+            <a class="nav-link" data-bs-toggle="collapse" href="#auth" aria-expanded="false" aria-controls="auth">
+              <i class="menu-icon mdi mdi-calendar-plus"></i>
+              <span class="menu-title">Add Event</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" data-bs-toggle="collapse" href="#auth" aria-expanded="false" aria-controls="auth">
+              <i class="menu-icon mdi mdi-calendar-multiple"></i>
+              <span class="menu-title">Events</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <!---------------------------------------------------Jobs Category------------------------------------------------->
+          <li class="nav-item nav-category">Jobs</li>
+          <li class="nav-item">
+            <a class="nav-link" data-bs-toggle="collapse" href="#auth" aria-expanded="false" aria-controls="auth">
+              <i class="menu-icon mdi  mdi-earth-box-plus"></i>
+              <span class="menu-title">Add Categories</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" data-bs-toggle="collapse" href="#auth" aria-expanded="false" aria-controls="auth">
+              <i class="menu-icon mdi mdi-earth-box"></i>
+              <span class="menu-title">Categories</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" data-bs-toggle="collapse" href="#auth" aria-expanded="false" aria-controls="auth">
+              <i class="menu-icon mdi  mdi-briefcase"></i>
+              <span class="menu-title">Jobs</span>
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <!------------------------------------------------------------------------------------------->
+          <li class="nav-item nav-category">help</li>
+          <li class="nav-item">
+            <a class="nav-link" href="http://bootstrapdash.com/demo/star-admin2-free/docs/documentation.html">
+              <i class="menu-icon mdi mdi-file-document"></i>
+              <span class="menu-title">Documentation</span>
+            </a>
+          </li>
+        </ul>
+      </nav>
+<!--------------------------------------------------------End of SideBar------------------------------------------------->
+
+      <!-- Table -->
```

Comment:
I don't like such long comments, please remove it

Reply:
this is for me because scrolling in long files gets my eyes crossed ,i will remove it before i upload next time